node-tests/fixtures/mixin-test/rfc232.js

```diff
@@ -0,0 +1,15 @@
+import EmberObject from '@ember/object';
+import FooMixin from 'my-app/mixins/foo';
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+
+module('Unit | Mixin | foo', function(hooks) {
+  setupTest(hooks);
```

Comment:
I don't think we necessarily need the `setupTest()` call here since we're not using the container for anything 🤔

Reply:
I will remove it.

Reply:
One more question. Should I squash the commits after that, ince it's a small change reverting some of this ?

Reply:
yes, I think that would be good, thanks! :)